MekHQ/src/mekhq/campaign/log/LogEntryController.java

```diff
@@ -168,49 +185,47 @@ private static boolean foundExpressionWithThreeVariables(String logEntryProperty
      * @param description to be changes
      * @return string with the new description.
      */
-    public static String updateOldDescription(String description){
+    public static String updateOldDescription(String description) {
         String newDescription = "";
 
         newDescription = updatePrisonerDescription(description);
```

Comment:
We can also move this up to the declaration and remove the meaningless initialization.

Reply:
Can do.